### DISCUSSION TITLE
making-drawing-circuits-in-markdown-a-cinch

### DISCUSSION THREAD

**Discussion by giscus** [engineerjoe440/stanley-solutions-blog#12] [Comments]
# making-drawing-circuits-in-markdown-a-cinch

I've talked about how someone can make PlantUML diagrams come to life, directly in markdown for blog-sites, and I've touched on other automation techniques I use to make blogs come together from plain text, but how about some circuit diagrams? Well, there wasn't a neat tool to help make this a possibility, until now!

https://blog.stanleysolutionsnw.com/making-drawing-circuits-in-markdown-a-cinch.html

**Comment by Naresh-git03** [comment]
+12V
                                 │
                     ┌───────────┴───────────┐
                     │                       │
                   Street                   Relay
                    Light                   Coil
                     │                       │
                     └────────────┬──────────┘
                                  │
                                 C│
                              ┌───────┐
                              │ BC547 │
                              └─┬─────┘
                                │E
                                │
                               GND

                     (Op-Amp Comparator LM358)

    +12V ────/\/\/──┐
           LDR      │
                    ▼
                  ┌───────────┐
                  │   +       │
                  │  LM358    │───── Output to transistor base
                  │   -       │
                  └────▲──────┘
                       │
         GND ────/\/\/─┘
               10k
